app/models/report.rb

```diff
@@ -64,7 +65,7 @@ def self.item_to_row(item)
   # get the average of an array of values, discounting nil values
   def self.average2(arr)
     arr = arr.reject(&:nil?)
```

Comment:
I'm going to address this comment since it's not really included in #1570 (as compared to most of the other comments) and then we should be good to go.